readthedocs/vcs_support/base.py

```diff
@@ -67,13 +72,7 @@ def __init__(
         self.verbose_name = verbose_name
         self.version_type = version_type
 
-        # TODO: always pass an explicit environment
-        # This is only used in tests #6546
-        #
-        # TODO: we should not allow ``environment=None`` and always use the
-        # environment defined by the settings
-        from readthedocs.doc_builder.environments import LocalBuildEnvironment
-        self.environment = environment or LocalBuildEnvironment()
+        self.environment = environment
```

Comment:
There are no changes in behavior, right? Because the default was to use `LocalBuildEnvironment` and all cases that didn't supply an environment now explicitly supply this one?

Reply:
For non-test code we are always passing an environment, this "feature" was only being used by our tests.

Reply:
Hehe, okay this change is maximally neutral then (if I may be allowed to conjugate "neutral")